app/components/vote-overview.ts

```diff
@@ -19,6 +20,47 @@ export default class VoteOverview extends Component<ArgsInterface> {
     return this.args.numberOfProponents || 0;
   }
 
+  get numberOfAbstentionsGraphValue() {
+    const totalValue =
+      (this.args.numberOfProponents || 0) +
+      (this.args.numberOfOpponents || 0) +
+      (this.args.numberOfAbstentions || 0);
+    const abstentionsValue = this.args.numberOfAbstentions || 0;
+
+    return (abstentionsValue / totalValue) * 158;
```

Comment:
What does the `158` mean exactly? Might be worth storing it in a descriptive const variable to make that more clear?

Reply:
Bit of a magic number. The length of the stroke-dasharray in a circle is 158. Will put it in a variable.